tests/generated/vk_enum_string_helper.cpp

```diff
@@ -5,5 +5,72 @@
 // SPDX-License-Identifier: Apache-2.0
 #include <vulkan/vk_enum_string_helper.h>
 
-// TODO: Add actual testing
-int main() {}
+#include <magic_enum.hpp>
+
+#include <gtest/gtest.h>
+
+TEST(vk_enum_string_helper, string_VkResult) {
+    constexpr auto values = magic_enum::enum_values<VkResult>();
+    for (auto val : values) {
+        auto magic_str = magic_enum::enum_name(val);
+
+        auto str = string_VkResult(val);
+
+        EXPECT_STREQ(magic_str.data(), str);
+    }
```

Comment:
https://github.com/Neargye/magic_enum#examples

magic_enum makes testing these functions quite easy.

Note I didn't add testing for each enum possible. I just picked a few significant ones. Since we are mainly testing the code generation above anything else.